MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include LICENSE.txt README.md requirements.txt requirements-dev.txt
 include rasa/core/schemas/* rasa/core/training/visualization.html
-include rasa/core/default_config.yml
+include rasa/cli/default_config.yml
```

Comment:
not sure why we need this change, the file is at https://github.com/RasaHQ/rasa/blob/master/rasa/core/default_config.yml

Reply:
ah so it is duplicated at the `rasa/cli/` path, and `rasa/constants.py` refers to it using that path: 
https://github.com/RasaHQ/rasa/blob/master/rasa/constants.py#L15

Should I change constants? in any case should remove one duplicate